### ISSUE TITLE
[Feature Card] focus state additional border

### ISSUE THREAD

**Issue by peter-garvin** [carbon-design-system/ibm-dotcom-library#2206]
This change is the result of design changes.  The designs have been updated with these changes.  

The border added on focus needs an additional 3px border, resulting in a 1px 'gap' between the blue focus border and the card. See image below
<img width="1076" alt="Screen Shot 2020-04-27 at 9 36 23 AM" src="https://user-images.githubusercontent.com/47458576/80384513-95d69380-886a-11ea-97e3-6228e6cf4857.png">

**Comment by RobertaJHahn** [comment]
@Praveen-sr @chetankan Could you have Ram take a look at this issue and help us determine what the priority should be?  Thanks.

**Comment by chetankan** [comment]
@RobertaJHahn : Sure will Ram to check.

**Comment by RobertaJHahn** [comment]
@RaphaelAmadeu Peter created another bug for this focus state border.  We have the fix assigned to the June release.  Would you be able to work on this before the code freeze, June 12?

cc:  @jeffchew
